tests/test_files.py

```diff
@@ -39,11 +47,11 @@ def test_WellFile__opens_and_get_user_account():
     wf = WellFile(
         os.path.join(
             PATH_OF_CURRENT_FILE,
-            "M120171010__2020_07_22_201922",
-            "M120171010__2020_07_22_201922__A1.h5",
+            "2020_08_04_build_775",
+            "MA20001010__2020_08_04_220041__D6.h5",
         )
     )
-    assert wf.get_user_account() == "bab42d5a-25ab-4b88-90ca-55914b55cf58"
+    assert wf.get_user_account() == "455b93eb-c78f-4494-9f73-d3291130f126"
```

Comment:
when our SDK loads this attribute, convert it to an actual python `UUID` object (same with other UUIDs)---makes it easier for comparisons and other functions down the road